articles/machine-learning/how-to-auto-train-image-models.md

```diff
@@ -359,14 +359,43 @@ In individual trials, you directly control the model architecture and hyperparam
 Object detection | **YOLOv5**: One stage object detection model   <br>  **Faster RCNN ResNet FPN**: Two stage object detection models  <br> **RetinaNet ResNet FPN**: address class imbalance with Focal Loss <br> <br>*Note: Refer to [`model_size` hyperparameter](reference-automl-images-hyperparameters.md#model-specific-hyperparameters) for YOLOv5 model sizes.*| ***`yolov5`\**** <br> `fasterrcnn_resnet18_fpn` <br> `fasterrcnn_resnet34_fpn` <br> `fasterrcnn_resnet50_fpn` <br> `fasterrcnn_resnet101_fpn` <br> `fasterrcnn_resnet152_fpn` <br> `retinanet_resnet50_fpn` 
 Instance segmentation | **MaskRCNN ResNet FPN**| `maskrcnn_resnet18_fpn` <br> `maskrcnn_resnet34_fpn` <br> ***`maskrcnn_resnet50_fpn`\****  <br> `maskrcnn_resnet101_fpn` <br> `maskrcnn_resnet152_fpn`
 
+With the new AutoML on Pipelines feature (preview), you can additionally use any image classification model from the [HuggingFace Hub](https://huggingface.co/models?pipeline_tag=image-classification&library=transformers) which are part of the transformers library (such as microsoft/beit-base-patch16-224), as well as any object detection or instance segmentation model from the [MMDetection Version 2.28.2 Model Zoo](https://mmdetection.readthedocs.io/en/v2.28.2/model_zoo.html) (such as atss_r50_fpn_1x_coco). Using any HuggingFace or MMDetection model will trigger runs using pipeline components. If both legacy and HuggingFace/MMdetection models are used, all runs/trials will be triggered using components. Curated models from HuggingFace and MMDetection are provided in the azureml-staging registry. These curated models have been tested thoroughly and use default hyperparameters selected from extensive benchmarking to ensure effective training. The table below summarizes these curated models
+
+Task |  model architectures
+---|---------
+Image classification<br> (multi-class and multi-label)| [**microsoft/beit-base-patch16-224-pt22k-ft22k**](https://ml.azure.com/registries/azureml-preview/models/microsoft-beit-base-patch16-224-pt22k-ft22k/version/1): The BEiT model is a Vision Transformer (ViT), which is a transformer encoder model (BERT-like). <br> [**google/vit-base-patch16-224**](https://ml.azure.com/registries/azureml-preview/models/google-vit-base-patch16-224/version/1): The Vision Transformer (ViT) is a transformer encoder model (BERT-like) pretrained on a large collection of images in a supervised fashion, namely ImageNet-21k, at a resolution of 224x224 pixels. <br> [**facebook/deit-base-patch16-224**](https://ml.azure.com/registries/azureml-preview/models/facebook-deit-base-patch16-224/version/1): This model is actually a more efficiently trained Vision Transformer (ViT). <br> [**microsoft/swinv2-base-patch4-window12-192-22k**](https://ml.azure.com/registries/azureml-preview/models/microsoft-swinv2-base-patch4-window12-192-22k/version/1): The Swin Transformer is a type of Vision Transformer.
```

Comment:
Let's have the same table format as above - with the additional colum for the string syntax, and only BEIT name in the first